### ISSUE TITLE
Permission errors upon first launch

### ISSUE THREAD

**Issue by fredrikscode** [autobrr/autobrr#923]
**Version**
latest

**Describe the bug**
Permission errors upon first launch

**To Reproduce**
1. Run compose through Portainer (vanilla Docker should be just the same though)
2. Check logs

**Expected behavior**
Logs look good and autobrr works :)

**Desktop (please complete the following information):**
 - OS: Arch
 - Browser: Brave
 - Version: 1.50.125

**Log**
```
2023/05/09 08:11:47 error creating file: "open /config/config.toml: permission denied"
2023/05/09 08:11:47 write error: "open /config/config.toml: permission denied"
2023/05/09 08:11:47 config read error: "open /config/config.toml: no such file or directory"
2023-05-09T08:11:47+02:00 fatal could not open sqlite db connection | module=database stack=[{"func":"(*DB).openSQLite","line":"37","source":"sqlite.go"},{"func":"(*DB).Open","line":"71","source":"database.go"},{"func":"main","line":"70","source":"main.go"},{"func":"main","line":"250","source":"proc.go"},{"func":"goexit","line":"1598","source":"asm_amd64.s"}] error=enable wal: unable to open database file: out of memory (14) 
```
**Permissions on folder (not manually touched)**
`drwxr-xr-x.  3 root  root    20 May  9 08:11 autobrr`

**Compose**
```
version: "3.7"

networks:
  portainer:
    external: true

services:
  autobrr:
    container_name: autobrr
    image: ghcr.io/autobrr/autobrr:latest
    restart: unless-stopped
    security_opt:
      - no-new-privileges:true
    #logging:
    #  driver: json-file
    #  options:
    #    max-file: ${DOCKERLOGGING_MAXFILE}
    #    max-size: ${DOCKERLOGGING_MAXSIZE}
    user: 1000:1000
    environment:
      - TZ=Europe/REDACTED
    networks:
      - portainer
    volumes:
      - /opt/docker/services/autobrr/config:/config
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.autobrr.entrypoints=https"
      - "traefik.http.routers.autobrr.rule=Host(`autobrr.REDACTED.io`)"
      - "traefik.http.routers.autobrr.tls=true"
      - "traefik.http.routers.autobrr.tls.certresolver=cloudflare"
      - "traefik.http.routers.autobrr.service=autobrr"
      - "traefik.http.middlewares.autobrr-redirect.redirectscheme.scheme=https"
      - "traefik.http.services.autobrr.loadbalancer.server.port=7474"
```
The service account on my server has id 1000 in this case.

**Comment by martylukyy** [comment]
Hi! 

Your autobrr folder is owned by the root (UID 0) user.
The user you are running the container with is the user with the id 1000.
Due to the permission for the folder being drwxr-xr`-`x the user with the id 1000
cannot write to the config.toml or write to the database.

Please try to recursively take ownership of your autobrr folder and start the docker container again.
`(sudo) chown -R 1000:1000 /path/to/autobrr`

**Comment by fredrikscode** [comment]
Hi!

Ah, I forgot that when letting Docker create folders by itself it'll default to root. 

`drwxr-xr-x. 3 admin admin 20 May  9 08:11 autobrr`
`drwxr--r--. 2 admin admin 87 May  9 12:42 config`
`-rwxr--r--. 1 admin admin 135168 May  9 12:42 autobrr.db`
`-rwxr--r--. 1 admin admin  32768 May  9 12:43 autobrr.db-shm`
`-rwxr--r--. 1 admin admin   8272 May  9 12:43 autobrr.db-wal`
`-rwxr--r--. 1 admin admin    915 May  9 12:42 config.toml`

After changing the permissions and restarting autobrr I was prompted to create an account, which I did. The account is unable to login though. I made it simple, admin/hello, but I still get notified of a wrong username/password.
